### DISCUSSION TITLE
[UX] sync duplication offset with grid size (and step size for moving object with keyboard)?

### DISCUSSION THREAD

**Discussion by ben-wes** [plugdata-team/plugdata#1705] [Ideas]
step size and grid size are synced, which feels right imo. it might be a nice addition if the grid size would also affect the duplication offset though - this would make it way easier to place objects at an aligned position via keyboard after duplication.

**Comment by timothyschoen** [comment]
Sounds good! I think we should only do this if the absolute grid is enabled. I think spacing it out by 25px feels like a too large jump, so that would only feel right if the grid is actually enabled, I think

**Reply by ben-wes** [reply]
ah, i see what you mean when testing this here now. i certainly agree that it's an option to just have this behaviour relate to the grid definition.

what i don't yet completely understand: how did the 25px step distance come about? was it mainly because 10px steps seemed too small (requiring too many keystrokes for creating proper layouts)?

**Reply by timothyschoen** [reply]
I think having a grid with 10px spacing looks too busy for me. Especially for people who don't use the absolute grid (where the grid dots mostly just help to distinguish between edit and lock mode), a 25px grid feels more appropriate.

**Reply by ben-wes** [reply]
understood. but would it be an option to default the arrow key's step size to 10px in that case (as is the case in puredata)? with the 10px offset on duplication and the 25px steps, it's a bit hard to align objects with keyboard interaction.
